bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/persistence/PersistenceItemInfoBean.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.io.rest.core.persistence;
+
+/**
+ * This is a java bean that is used to serialize information about items in a persistence service to JSON.
+ *
+ * @author Chris Jackson - Initial Contribution
+ *
+ */
+public class PersistenceItemInfoBean {
```

Comment:
Note that we by now always call such beans DTOs and where ever they might be relevant for serialization anywhere else than the REST API as well (e.g. for the payload of events), we put those DTOs in the core bundle in a sub-package "dto" (here it would be org.eclipse.smarthome.core.persistence.dto).

Could you possibly refactor the existing "Beans" in this package to follow this convention?
Regarding the PersistenceItemInfoBean: Do we need this at all? It is very similar to the PersistenceItemInfo class - maybe you could add a simply GSON annotation to the Date fields to specify the string format in which they should be serialized?